server/revbump/revbump.go

```diff
@@ -0,0 +1,65 @@
+package revbump
+
+import (
+	"go.etcd.io/etcd/server/v3/mvcc"
+	"go.etcd.io/etcd/server/v3/mvcc/backend"
+	"go.etcd.io/etcd/server/v3/mvcc/buckets"
+	"go.uber.org/zap"
+)
+
+func UnsafeModifyLastRevision(lg *zap.Logger, bumpAmount uint64, be backend.Backend) error {
+	defer be.ForceCommit()
+
+	tx := be.BatchTx()
+	tx.LockOutsideApply()
+	defer tx.Unlock()
+
+	latest, err := unsafeGetLatestRevision(tx)
+	if err != nil {
+		return err
+	}
+
+	latest = unsafeBumpRevision(lg, tx, latest, int64(bumpAmount))
+	unsafeMarkRevisionCompacted(lg, tx, latest)
+	return nil
+}
+
+func unsafeBumpRevision(lg *zap.Logger, tx backend.BatchTx, latest revision, amount int64) revision {
+	lg.Info(
+		"bumping latest revision",
+		zap.Int64("latest-revision", latest.main),
+		zap.Int64("bump-amount", amount),
+		zap.Int64("new-latest-revision", latest.main+amount),
+	)
+
+	latest.main += amount
+	latest.sub = 0
+	k := make([]byte, 17)
```

Comment:
nit: Could do revBytesLen instead of 17

Reply:
good catch, I was just copy and pasting it over to make the dependency graph happy

Reply:
fixed